packages/components/src/molecules/CartItem/CartItem.tsx

```diff
@@ -0,0 +1,100 @@
+import type { HTMLAttributes, ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+import { XCircle } from '../../assets'
+import {
+  IconButton,
+  IconButtonProps,
+  Price,
+  QuantitySelector,
+} from '../../index'
+
+export type PriceVariant =
+  | 'selling'
+  | 'listing'
+  | 'spot'
+  | 'savings'
+  | 'installment'
+
+export type PriceFormatter = (price: number, variant: PriceVariant) => ReactNode
```

Comment:
I think we can import this  { PriceFomatter }  form Price component. So we don't need to define `PriceFomatter` and the `PriceVariant` types.